Documentation/Links.md

```diff
@@ -0,0 +1,59 @@
+-   [SerenityOS on English Wikipedia](https://en.wikipedia.org/wiki/SerenityOS). This page also links to the SerenityOS articles in many other languages.
+
+## Development
+
+-   [GitHub Organization](https://github.com/SerenityOS) and [GitHub Repositories](https://github.com/orgs/SerenityOS/repositories)
+-   [Changelog](https://changelog.serenityos.org/)
+-   [Issues Found by OSS-Fuzz Continous Fuzzing](https://bugs.chromium.org/p/oss-fuzz/issues/list?q=label:Proj-serenity)
+-   [Azure CI Overview](https://dev.azure.com/SerenityOS/SerenityOS/_build)
+-   [SonarCloud Static Analysis](https://sonarcloud.io/project/overview?id=SerenityOS_serenity)
+-   [libjs.dev](https://libjs.dev/)
+    -   [Try LibJS!](https://libjs.dev/repl/)
+-   [Compiler Explorer](https://serenity.godbolt.org/): Select "Lagom trunk" under "Libraries" and add the compiler option `-std=c++20`
+-   [LibFont.js](https://macdue.github.io/LibFont.js/)
+
+## Social
+
+-   [Discord Server](https://discord.gg/serenityos)
+-   [serenityos.social](https://serenityos.social/), the official Mastodon instance
+-   [Map of Developers and Users](https://usermap.serenityos.org/)
+-   YouTube Channel of [Andreas Kling](https://www.youtube.com/@awesomekling), [Linus Groh](https://www.youtube.com/@linusgroh), [kleines Filmröllchen](https://www.youtube.com/@kleinesfilmroellchen), and [davidot](https://www.youtube.com/@davidot4475)
+-   Personal Blog of [Andreas Kling](https://awesomekling.github.io/), [Linus Groh](https://linus.dev/posts/), [Andrew Kaster](https://adkaster.github.io/), [Jesse Buhagiar](https://quaker762.github.io/), and [Sam Atkins](https://atkinssj.github.io/)
+
+## Statistics and Lists
+
+-   [:^) Tracker](https://happy-serenityos.linus.dev/)
+-   [FIXMEs](https://benwiederhake.github.io/serenity-fixmes/)
+    -   [Flamegraph Distribution](https://benwiederhake.github.io/serenity-fixmes/flamegraph.html)
+-   [Lines of Code vs. Commits](https://github.com/alimpfard/random-serenity-statistics#random-serenityos-statistics)
+-   [CSS Conformance](https://css.tobyase.de/)
+-   [Fonts](https://fonts.serenityos.net/)
+-   [Emoji Table](https://emoji.serenityos.net/)
+    -   [Emoji Statistics](https://emoji.serenityos.net/chart.emoji.serenityos)
+-   [Flags](https://flags.serenityos.net/)
+-   [test262](https://libjs.dev/test262/) (JavaScript Spec Tests)
+-   [Wasm Spec Tests](https://libjs.dev/wasm/)
+-   [Test Performance](https://github.com/alimpfard/random-serenity-statistics/tree/main/view/benchmarks/x86_64)
+-   [serenityos.social Statistics](https://grafana.serenityos.social/public-dashboards/42f9a11380614367aa4851076ee6a66b?orgId=1)
+
+## Miscellaneous
+
+-   [Merch](https://teespring.com/stores/serenityos?utm_medium=product_shelf&utm_source=youtube&utm_content=YT-AGNkefCob-U_Mq4OnxSUagj77Nz8jYYCc-IuarARfod3YK0765pOaAwNayHAn5Mu6YTyuINDxjU33HWU_eb8Njg6x-XlSo_VxfBNcJFUWOheXkivDRfjrEn-p8J3GFJO7uh8T0qUoWqtdlJHviuwc1AdLRbDbltBe6VuFWlLFWrhihhUk0XmjQ%3D%3D)
```

Comment:
We should probably strip that metadata from the link. Also, don't we have `https://store.serenityos.org`?